Source/RuntimeMeshComponent/RuntimeMeshComponent.Build.cs

```diff
@@ -8,20 +8,21 @@ public RuntimeMeshComponent(ReadOnlyTargetRules rules) : base(rules)
     {
         PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
 
-        PublicIncludePaths.AddRange(
-            new string[] {
-                "RuntimeMeshComponent/Public"
-				// ... add public include paths required here ...
-			}
-            );
-
-
-        PrivateIncludePaths.AddRange(
-            new string[] {
-                "RuntimeMeshComponent/Private",
-				// ... add other private include paths required here ...
-			}
-            );
+        // HORU: this was throwing warnings
```

Comment:
In both `.Build.cs` files you can fix this portion by adding `using System.IO;` and then
```
PrivateIncludePaths.AddRange(new string[] { Path.Combine(ModuleDirectory, "Private") });
PublicIncludePaths.AddRange(new string[] { Path.Combine(ModuleDirectory, "Public") });
```

It seems to work fine in both cases, but this should be the proper way.

Reply:
I ended up making changes to this version while updating another plugin. Rather than make a competing PR I just submitted it to this fork. Minor cleanup and support for compiling in older engine versions. (https://github.com/GlassBeaver/RuntimeMeshComponent/pull/2)

Reply:
merged, thank you!